### ISSUE TITLE
Large spray surface area check is unnecessary

### ISSUE THREAD

**Issue by sappykun** [nm004/img2tempdecal#1]
The surface area check for Goldsrc sprays appears to be unnecessary.  I am able to load sprays with a surface area of 14336 in other Goldsrc titles such as Counter-Strike 1.6 (at least in Goldsrc builds 9920 (current) and 8284 (legacy), I haven't checked with anything earlier than that), so the toggle should be removed and the surface size should always be set to 14336.

For example, here is a spray I have made (in PNG format) with dimensions 112x128:

![}LOGO](https://github.com/nm004/img2tempdecal/assets/37008692/5597eae2-4545-4049-9e54-2e517d4a76a8)

[tempdecal.zip](https://github.com/nm004/img2tempdecal/files/14967744/tempdecal.zip)

and here it is in Counter Strike:

![20240413090106_1](https://github.com/nm004/img2tempdecal/assets/37008692/17c31844-ce94-4aed-a976-c7ec1a2541a3)

I believe this misconception started since HLTagConverter wasn't able to process images larger than 12288 for some reason, so everyone assumed that was the maximum size.  A very similar thing happened with Source engine sprays, where everyone assumed spray dimensions had to be a multiple of 2 because VTFEdit insisted on it when you imported images (it was trying to create mipmaps for textures, which DO require 2^n dimensions, but DXT textures only need to have their dimensions be a multiple of 4 if you have exactly one mipmap).

**Comment by nm004** [comment]
I agree with your opinion. I will remove the option in the next release.

Let me explain why I added the option for my future notes. As of time of writing the code, I refered to [the 303's GoldSrc Spraylogo Tutorial](http://the303.org/tutorials/goldsrcspraylogo.html) as the one of the reference about Goldsrc spray format. It says "Valid sizes ONLY for Sven Co-op using manual .wad creation in wally.exe (maximum surface 14436)". However, as @sappykun posted it, large size spray can be used in Half-Life and its mods.

I could use your spray with the newer build of Goldsrc (25th anniversary update) as you did. I tested it on the vanilla Half-Life, [Half-Life: Before, which is available on Steam](https://store.steampowered.com/app/261980/HalfLife_Before/) and [Scientist Slaughterhouse, which is available on RSTL](https://www.runthinkshootlive.com/posts/scientist-slaughterhouse/). The spray could be used on all of them.

Anyway, thank you for your report.
